### PR TITLE
add test on zone types

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: tests/cosmogony_lux_test.rs [hunk 1]
```diff
@@ -87,3 +87,26 @@ fn test_lux_admin_levels_with_serialisation() {
...
     test_wrapper_for_lux_admin_levels(cosmogony_from_json);
 }
+
+#[test]
+fn test_lux_zone_types() {
+    // Check the zone types in the built cosmogony
+    let cosmogony = create_cosmogony_for_lux();
+    let zone_type_counts = cosmogony.meta.stats.zone_type_counts;
+
+    fn assert_count(counts: &BTreeMap<String, u64>, key: String, value: u64) {
```
Comment (amatissart) [inline tests/cosmogony_lux_test.rs]: A very small detail, but we could directly define `key` as a `&str` here, and remove all the `format!` below.

File: tests/cosmogony_lux_test.rs [hunk 2]
```diff
@@ -87,3 +87,26 @@ fn test_lux_admin_levels_with_serialisation() {
 
     test_wrapper_for_lux_admin_levels(cosmogony_from_json);
 }
+
+#[test]
+fn test_lux_zone_types() {
+    // Check the zone types in the built cosmogony
+    let cosmogony = create_cosmogony_for_lux();
```
Comment (antoine-de) [inline tests/cosmogony_lux_test.rs]: even if that's way better to separate the tests, isn't the cosmogony rebuilding a bit too long ?

Rust definitely lacks a good py.test like framework 😜

Comment (nlehuby) [reply tests/cosmogony_lux_test.rs]:
```
real	0m10.105s
user	0m30.044s
sys	0m0.508s
```

File: tests/cosmogony_lux_test.rs (hunk 1 above)
Comment (nlehuby) [reply tests/cosmogony_lux_test.rs]: :+1:

**Pull request merged**